### PR TITLE
Spent timestamp

### PR DESCRIPTION
**Pull request by tdroxler**
As explain in [this issue](https://github.com/alephium/explorer-backend/issues/86)

A Release will need to be done and run over our network before the next one.

### CODE REVIEW

File: app/src/main/scala/org/alephium/explorer/service/TokenSupplyService.scala [hunk 1]
```diff
@@ -223,12 +222,15 @@ case object TokenSupplyService extends TokenSupplyService with StrictLogging {
...
-              .result
-              .headOption
+            sql"""
+              SELECT block_timestamp
+              FROM block_headers
+              WHERE main_chain = true
+              AND chain_from = $from
+              AND chain_to = $to
+              ORDER BY block_timestamp DESC
+              LIMIT 1
```
Comment (simerplaha) [inline app/src/main/scala/org/alephium/explorer/service/TokenSupplyService.scala:232]: The typed query being replaced is not querying for `main_chain = true`. Is the `main_chain` check in SQL query something that was missing from the typed query?

Comment (tdroxler) [reply app/src/main/scala/org/alephium/explorer/service/TokenSupplyService.scala:232]: Good point, I think it actually doesn't change much with or without it, token supply is computed daily at midnight, the latest  block is more like a time reference to see when we change day. 
As it's the same it's maybe cleaner to have the `main_chain = true`?

**Review by simerplaha — APPROVED** [review]
👍🏼

**Review by polarker — APPROVED** [review]
LGTM

**Comment by tdroxler** [comment]
I'll make a release on Monday and deploy it on our various env to let the new field spread out.

**Pull request merged**